### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse – New Feature_Storage Container_Keyboard: In Azure explorer tab, user cannot be able access ‘Jobs’ using keyboard and unable to expand the cluster unless user refreshes the cluster

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3556]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #MAS2.1.1; #Keyboard; #C+E-H; #JavatoolingforIntelliJ/Eclipse

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS2.1.1

**Repro Steps:** 
1) Open Java EE - Eclipse application
2) Install 'Azure Toolkit for Eclipse' from Marketplace
3) Open Eclipse and install the Scala plug-in 
4) Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
5) Tab till Windows menu option-->Show View-->Other-->Expand Azure-->Azure Explorer pane press OK
6) In Azure Explorer section, tab till HDInsight and expand it. Expand the cluster name and select Jobs
7) Verify if user can be able to expand and select the Jobs with the help of keyboard

**Actual Result:** 
In Azure explorer tab, user cannot be able access ‘Jobs’ using keyboard and unable to expand the HDInsight menu items and the Cluster details unless he presses right click and refresh each menu item

**Expected Result:** 
When user presses enter or space on HDInsight and on the clusters or the Jobs it should get expanded and get accessed with the help of keyboard

**User Impact:** 
Keyboard users will not be able to select the cluster options such as Jobs and storage accounts if they cannot be accessed with the help of keyboard

**MAS Reference:** 
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

[Bug_2_user is unable access jobs using keyboard.pptx](https://github.com/microsoft/azure-tools-for-java/files/3635504/Bug_2_user.is.unable.access.jobs.using.keyboard.pptx)

**Comment by t-rufang** [comment]
Actually you can use the "Apps" key in the keyboard to trigger the menu on a node. The same as #3607 
![image](https://user-images.githubusercontent.com/32627233/67483923-7717fa00-f699-11e9-8430-2093ce556420.png)

**Comment by t-rufang** [comment]
Previously in Eclipse we can't use keyboard to open HDInsight jobs view.
With #3772, we can use `Enter` key to trigger jobs view.

**Comment by Tanushagottipati** [comment]
Regressed in the below version and the issue is fixed. Hence closing the bug and attached latest video for reference
Environment Details:
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.19.0

[jobs issue fixed.pptx](https://github.com/microsoft/azure-tools-for-java/files/3878083/jobs.issue.fixed.pptx)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749666
